src/tera_main.cc

```diff
@@ -78,6 +77,9 @@ int main(int argc, char** argv) {
         signal(SIGTERM, SignalIntHandler);
         // signal(SIGSEGV, SIG_DFL); // 如果这个被改回SIG_DFL, jvm会core, 不知道为啥...
     }
+    if (g_quit) {
```

Comment:
How do you think about to replace the previous statement “`break;`” by “`goto Shutdown;`” so that a duplicate check could be avoided for the variable “`g_quit`”?